analyzers/nodejs/nodejs_discover_test.go

```diff
@@ -0,0 +1,48 @@
+package nodejs_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers/nodejs"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+/* Nested Module Order
+    └─┬ nested-modules
+      └─┬ module-1
+	  ├── module-2
+	  └── module-3
+*/
+func TestNestedModules(t *testing.T) {
```

Comment:
were these tests failing before you made the logic change? I'm not seeing how this tests the trailing `/` issue

Reply:
This test doesn't specifically test for trailing `/`'s or analyze's reaction to them. What this does test for is that we are correctly building .fossa.yml files. I decided to implement this after talking with @liftM to prevent future degrading changes.